src/main/resources/assets/admin/common/js/ui/time/Picker.ts

```diff
@@ -12,16 +12,24 @@ import {SelectedDateChangedEvent} from './SelectedDateChangedEvent';
 export class Picker<T extends Element>
     extends DivEl {
 
+    private showDefaultButton: Boolean = false;
```

Comment:
`boolean` is a primitive type. But you don't need this extra property, `defaultValueHandler` is enough.